textfsm_test.py

```diff
@@ -846,5 +857,27 @@ def testFillup(self):
         str(result))
 
 
+  def testRepeated(self):
+    """Repeated option should work ok."""
+    tplt = """Value Repeated repeatedKey (\S+)
+Value Repeated repeatedValue (\S+)
+Value normalData (\S+)
+Value normalData2 (\S+)
+Value Repeated repeatedUnused (\S+)
+
+Start
+  ^${normalData} (${repeatedKey}:${repeatedValue} )*${normalData2} -> Record"""
+
+    data = """
+normal1 key1:value1 key2:value2 key3:value3 normal2 \n
```

Comment:
Maybe add a second line to show they are additive

Reply:
Similar to our discussion re: interaction between Repeated and List.

With input like this ...
"""
First record:  a1:b1, a2:b2, a3:b3
                       a4:b4, a5:b5
Second record: x1:y1, x2:y2
"""
I imagine the desired output would be:
First, [[a1, a2, a3, a4, a5], [x1, x2]], [[b1, b2, b3, b4, b5], [y1, y2]]

Where the first record is picked up over two lines. Which seems do-able and a complex example like this might help.

Reply:
Tests added in 0707470. Working on similar tests for Repeated with Filldown, Fillup, etc.